test/tests/functional/pbs_job_task.py

```diff
@@ -90,8 +92,11 @@ def test_singlenode_pbs_tmrsh(self):
         if job_status:
             job_output_file = job_status[0]['Output_Path'].split(':')[1]
 
-        with open(job_output_file, 'r') as fd:
-            job_out = fd.read().strip()
-            self.logger.info("job_out=%s" % (job_out,))
-
-        self.assertEqual(job_out, "OK")
+        ret = self.du.cat(hostname=self.mom.shortname,
+                          filename=job_output_file,
+                          runas=TEST_USER)
+        _msg = "cat command failed with error: %s" % ret['err']
+        self.assertEqual(ret['rc'], 0, _msg)
+        _msg = 'Job\'s error file has error: "%s"' % ret['out']
+        self.assertEqual(ret['out'][0], "OK", _msg)
+        self.logger.info("Job has executed without any error")
```

Comment:
please make a single function for checking file error and call that function with filename.